tracer/src/Datadog.Trace.Tools.dd_dotnet/Checks/ProcessConfiguration.cs

```diff
@@ -43,8 +43,8 @@ internal static IConfigurationSource ExtractConfigurationSource(this ProcessInfo
             }
         }
 
-        var jsonConfigurationSource = JsonConfigurationSource.TryLoadJsonConfigurationFile(configurationSource, baseDirectory);
-
+        var jsonConfigurationSource = JsonConfigurationSource.TryLoadJsonConfigurationFile(configurationSource, baseDirectory, out string jsonDirectory);
+        Utils.WriteInfo("Trying to load JSON configuration file from: " + jsonDirectory);
         if (jsonConfigurationSource != null)
         {
             configurationSource.Add(jsonConfigurationSource);
```

Comment:
thoughts? Just to only log it out when we do get a JSON file that we are loading

```suggestion
        var jsonConfigurationSource = JsonConfigurationSource.TryLoadJsonConfigurationFile(configurationSource, baseDirectory, out string jsonDirectory);
        if (jsonConfigurationSource != null)
        {
            Utils.WriteInfo("Loaded JSON configuration file from: " + jsonDirectory);
            configurationSource.Add(jsonConfigurationSource);
```

Reply:
@bouwkast - I think logging regardless if there is a json loaded is better. This is because the customers won't know if they misplaced the `datadog.json` folder.

My logic based on customer calls: 

Customer places json in the wrong folder -> no output since json is not being read -> not sure why it's not working
Customer places json in the wrong folder -> outputs that we tried to load the json from `c:\whatever\directory` -> customer moves the json to that directory and tries again.

Reply:
Ahh gotcha makes sense,

thoughts then on adding a specific log message then for the `if (jsonConfigurationSource != null)` and an additional `else` branch to say that no `datadog.json` was found?

Like:

```csharp
if (jsonConfigurationSource != null)
{
    Utils.WriteInfo("Loaded JSON configuration file from: " + jsonDirectory);
    configurationSource.Add(jsonConfigurationSource);
}
else
{
    Utils.WriteInfo("Did not find JSON configuration file to load");
}

```

Reply:
oh I like that. This could help in the case that they're loading a json configuration but they're not expecting to.
"I don't have this env var set and I'm not using anything else to set configs"  
"Log says you're using a datadog.json"
"oops idk where that came from"